docs/conf.py

```diff
@@ -87,7 +87,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_symbiflow_theme"
+html_theme = "sphinx_book_theme"
```

Comment:
@dralabeing looks like the issue with the headers being hidden are actually the expected behavior of the theme. Should we change the theme?

Reply:
Here is a quick list of things to consider (this is not a final or prioritized list but nice to have):
1. MyST MD elements - for table alignment- https://sphinx-book-theme.readthedocs.io/en/stable/reference/special-theme-elements.html#myst-markdown-elements
2. The use of full-width admonitions to add emphasis- perhaps code blocks
3. Sphinx extensions like

- tabbed content- https://sphinx-book-theme.readthedocs.io/en/stable/reference/extensions.html#sphinx-tabs-tabbed-content
-  Link to tutorial videos- https://sphinx-book-theme.readthedocs.io/en/stable/reference/extensions.html#sphinxcontrib-youtube-for-videos
- Support for Jupyter style otebooks- https://sphinx-book-theme.readthedocs.io/en/stable/reference/notebooks.html

Reply:
> There is a lot to read on this page. I will look at it and make suggestions. However, we don't want this to be a huge OR doc sytlization effort. The main request was that the right and left panels sync and also navigation is not multi-stepped.
> 
> @luarss Feel free to suggest them changes that will ease navigation as the first order and cleanup of the panel based navigation

So the theme change will help solve the main request. To clarify "navigation not multi-stepped", do you mean you wish to see more dropdown depth in the left nav bar? For example depth 3 will give this nav bar effect

- Getting Started with OpenROAD flow
-     Build With Docker
-           Prerequisites
- Clone and Build
-             Verify Installation
-              Enable GUI support
-             Build Docker Image for Different OS

> Here is a quick list of things to consider (this is not a final or prioritized list but nice to have):
> 
> 1. MyST MD elements - for table alignment- https://sphinx-book-theme.readthedocs.io/en/stable/reference/special-theme-elements.html#myst-markdown-elements
> 2. The use of full-width admonitions to add emphasis- perhaps code blocks
> 3. Sphinx extensions like
> 
> * tabbed content- https://sphinx-book-theme.readthedocs.io/en/stable/reference/extensions.html#sphinx-tabs-tabbed-content
> * Link to tutorial videos- https://sphinx-book-theme.readthedocs.io/en/stable/reference/extensions.html#sphinxcontrib-youtube-for-videos
> * Support for Jupyter style otebooks- https://sphinx-book-theme.readthedocs.io/en/stable/reference/notebooks.html
Admonitions is definitely one thing I want to add in, but I am not sure if it can be done for markdown as most tutorials use rst. Alternative is to use emojis in Markdown: 
![image](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/39641663/ffbbdca2-2be8-4d79-b3da-95aa69b9391e)

Reply:
@luarss Adding depth of navigation is useful as long as one can access it easily from the index i.e left bar into the desired topic. e.g the ability to click and be able to go to Verify Installation without having to scroll from from the top-bar i.e Getting Started.
Let's leave the theme extensions (admonitions etc) and stick to just ease of navigation and granularity in the docs.